src/data/python-quiz.ts

```diff
@@ -1531,6 +1531,15 @@ const pythonQuiz = [
     Explanation: "print('%.3f'%x) would round 297.4887 to 3 decimal places.",
     Link: "https://www.freecodecamp.org/news/2f-in-python-what-does-it-mean/",
   },
+   {
+    Question: "What is the result of math.trunc(3.1)?",
+    Answer: "3",
+    Distractor1: "3.0",
+    Distractor2: "3",
```

Comment:
distractors should be for wrong answers only

```suggestion
    Distractor2: "3.14",
```